### DISCUSSION TITLE
BarChart CharLabels->Placed not working as expected

### DISCUSSION THREAD

**Discussion by gganapat** [WLJSTeam/wljs-notebook#178] [General]
Hi:
I'm trying to place ChartLabels in specific position like Top since the default option is not rendering too well. So, my default version looks like this:

<img width="721" alt="Screenshot 2024-05-23 at 11 30 23 AM" src="https://github.com/JerryI/wolfram-js-frontend/assets/46061218/437c37c6-ead5-40ee-9e35-745eb9bafabd">

But when I try to place the labels at the top, I get this:

<img width="847" alt="Screenshot 2024-05-23 at 11 30 51 AM" src="https://github.com/JerryI/wolfram-js-frontend/assets/46061218/98c2366c-052b-4855-9a38-ff74603517c3">

Any help would be appreciated.

Thx!
Gani -

**Comment by JerryI** [comment]
#181

**Reply by JerryI** [reply]
Fixed!
